src/kata7.getEmployerRole.js

```diff
@@ -1,5 +1,9 @@
 const getEmployerRole = (employeeName, employees) => {
-
-}
+    for (let i = 0; i < employees.length; i++) {
```

Comment:
instead of using a for loop, you can also use the `.find` method on Array prototype.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find